### PR TITLE
Fixed typo in image, missing comma in header and removed some duplicated css

### PR DESCRIPTION
**Pull request by ichim-david**

### CODE REVIEW

**Comment by ichim-david** [comment]
@sneridagh I don't know if any of you get notified by community pull requests, this is what I see on my pull request right now.

**Comment by davisagli** [comment]
@ichim-david I approved the workflow to run -- that's a one-time approval for a new contributor, so you shouldn't see it again. Probably we can make the check less strict, I need to double check.

File: news/264.bug
Comment (davisagli) [inline news/264.bug:1]: I think you need to rename the extension to `.bugfix`

**Comment by ichim-david** [comment]
> @ichim-david I approved the workflow to run -- that's a one-time approval for a new contributor, so you shouldn't see it again. Probably we can make the check less strict, I need to double check.

@davisagli done, renamed also in this pull request as I made the same mistake with the news naming https://github.com/kitconcept/volto-light-theme/pull/268

**Comment by davisagli** [comment]
@ichim-david Okay, I think I got the CI stuff sorted out so that the workflows run even if you submit a PR from a fork.

**Review by davisagli — APPROVED** [review]
LGTM

**Pull request merged**